build/schema.js

```diff
@@ -13,21 +13,27 @@
             "properties": {
                 "alphabet": {
                     "type": "string",
-                    "minlength": 4,
-                    "maxlength": 4
+                    "pattern": "^[A-Z]{1}[a-z]{3}$"
                 },
                 "continent": {
-                    "type": "string",
-                    "minimum": 4,
-                    "maximum": 4
+                    "anyOf": [{
+                        "type": "string",
+                        "pattern": "^[A-Z]{2}$"
+                    }, {
+                        "type": "array",
+                        "items": {
+                            "type": "string",
+                            "pattern": "^[A-Z]{2}$"
+                        }
+                    }]
                 },
                 "language": {
                     "type": "string",
-                    "minlength": 1
+                    "pattern": "^[A-Za-z]+$"
```

Comment:
Hmmm...

* According to the Wikipedia [German Language page](https://en.wikipedia.org/wiki/German_language), they are calling it a "standard", e.g https://en.wikipedia.org/wiki/German_Standard_German, https://en.wikipedia.org/wiki/Austrian_Standard_German and
https://en.wikipedia.org/wiki/Swiss_Standard_German (this page mentions "regional variation").
* The [IETF](https://en.wikipedia.org/wiki/IETF_language_tag) calls them "subtags".

To me, it would make it *most* clear to label it as "region" or "variant".